backends-velox/src/test/scala/io/glutenproject/execution/VeloxFunctionsValidateSuite.scala

```diff
@@ -389,4 +389,22 @@ class VeloxFunctionsValidateSuite extends VeloxWholeStageTransformerSuite {
       checkOperatorMatch[ProjectExecTransformer]
     }
   }
+
+  test("Test unbase64 function") {
+    withTempPath {
+      path =>
+        Seq(
+          "U3BhcmsgU1FM"
+        )
+          .toDF("e")
+          .write
+          .parquet(path.getCanonicalPath)
+
+        spark.read.parquet(path.getCanonicalPath).createOrReplaceTempView("unbase64_tbl")
+
+        runQueryAndCompare("SELECT unbase64(e) from unbase64_tbl") {
+          checkOperatorMatch[ProjectExecTransformer]
+        }
+    }
+  }
```

Comment:
maybe use such way
```java
withTable("t") {
      withSQLConf("gluten disable") {
        sql("create table && write base64 string from normal string")
      }
      withSQLConf("gluten enable") {
        runQueryAndCompare("select unbase64") {
          checkOperatorMatch[FileSourceScanExecTransformer]
        }
      }
    }
```

Reply:
It looks great, but I have a question. Why should we consider gluten disabled? 
Maybe we can directly like this? Thanks for your guidance!
> withTable("t") {
        sql("create table && write base64 string from normal string")
        runQueryAndCompare("select unbase64(xx) from t") {
          checkOperatorMatch[ProjectExecTransformer]
        }
    }

Reply:
you are right.